transport/control.go

```diff
@@ -56,43 +56,33 @@ type windowUpdate struct {
 	increment uint32
 }
 
-func (windowUpdate) isItem() bool {
-	return true
-}
+func (*windowUpdate) isItem() {}
```

Comment:
Sorry, what?

As explained in the commit message, value receiver interface
implementations implement the interface for both value and pointer. In this
case, all the existing instances already use pointers, which is why there
is no fallout.
On Mar 7, 2016 16:36, "Brad Fitzpatrick" <notifications@github.com> wrote:

> In transport/control.go
> <https://github.com/grpc/grpc-go/pull/588#discussion_r55276401>:
>
> > @@ -56,43 +56,33 @@ type windowUpdate struct {
> >  	increment uint32
> >  }
> >
> > -func (windowUpdate) isItem() bool {
> > -	return true
> > -}
> > +func (*windowUpdate) isItem() {}
>
> But you would have to adjust the call sites too, which you don't. So this
> change is does not even seem correct.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc-go/pull/588/files#r55276401>.
>